lib/BaseController.js

```diff
@@ -127,6 +127,13 @@ module.exports = class BaseController {
   }
 
   errorHandler(err) {
+    if (`${err}` === '[object Object]') {
```

Comment:
```suggestion
    if ('object' === typeof err) {
```

Reply:
for some reason i thought stringify couldnt handle most JS objects, but looks like it can.